_includes/main.scss

```diff
@@ -105,6 +106,13 @@ input[type="submit"] {
   }
 }
 
+html, body { min-height: 100vh; }
+
+body > footer {
+  position: sticky;
+  top: 100vh;
+}
```

Comment:
Based on the approach documented here: https://css-tricks.com/a-clever-sticky-footer-technique/

Reply:
hmmm. I think I'd rather apply a flex to both the body and the footer. Sticky becomes problematic if the css for the content isn't consistent/allows for it.